pyproject.toml

```diff
@@ -58,7 +58,7 @@ select = [
 #    "C90",  # mccabe
 #    "I",  # isort
     "N",  # pep8-naming
-#    "UP",  # pyupgrade
+    "UP",  # pyupgrade
```

Comment:
Is it possible to control the base Python? The code must stay compatible with Python 3.6.

Reply:
Unfortunately ruff uses global Python version settings for all it's check, so while `pyupgrade` supports 3.6, ruff only goes to 3.7.  
I only found one instance of 3.6 compatibility issue and added `noqa` for that line. (subprocess.run universal_newlines)

Reply:
> I'd guess it reads either target-version or python_version but both are set to '3.7'. We need to have this as 3.6

Should be target-version in [tool.ruff], which only accepts py37+ see for example https://github.com/charliermarsh/ruff/issues/3826 This is already in the main, the question is whether to enable pyupgrade or not.

Reply:
Will disable the check then.  
Now, would it be ok to leave the changes, as they are all 3.6+? Or it's not worth to do code-style changes now, if they're not part of enabling a lint rule?